packages/template-retail-react-app/app/utils/utils.js

```diff
@@ -292,3 +292,34 @@ export const resolveLocaleFromUrl = (url) => {
         (locale) => locale.alias === defaultLocale || locale.id === defaultLocale
     )
 }
+
+const toCamel = (str) => {
```

Comment:
```suggestion
const safeToCamel = (str) => {
```
Or some other name, because it's not _strictly_ `toCamel`, as we don't modify all keys.